### PR TITLE
RemoteNotification should search for values in "aps" dictionary instead of top level

### LINKED ISSUE #6: RemoteNotification should search for values in "aps" dictionary instead of top level.
I'm not really sure about this, as I'm new to using remote notifications.
But reading the [documentation](https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/TheNotificationPayload.html) gives me the impression that the `userInfo` dictionary has an `aps` dictionary inside with the values.

So the RemoteNotification's init should actually look more like this:
```
public init?(remoteNotification: [NSObject : AnyObject]?) {
    guard let remoteNotification = remoteNotification as? [String : AnyObject] else {
        return nil
    }
    
    let apsKey = "aps"
    let aps = remoteNotification[apsKey] as? [String : AnyObject]
    
    alert = aps.flatMap(Alert.init(remoteNotification:))
    badge = aps?[badgeKey] as? Int
    sound = aps?[soundKey] as? String
    contentAvailable = aps?[contentAvailableKey] != nil
    categoryIdentifier = aps?[categoryKey] as? String
    remoteNotificationDictionary = remoteNotification
    
    var customFields = remoteNotification
    customFields.removeValueForKey(apsKey)
    
    userInfo = customFields
}
```

At least that seems to work for me, what do you think?

### PR DESCRIPTION
**Pull request by dfed**
cc @pwesten, @shawnwelch, @natan

Thanks @NinoScript for catching this!

Fixes #6

### CODE REVIEW

**Comment by dfed** [comment]
CI is having issues installing Cocoapods... working on that (fyi @dnkoutso). Tests run fine locally.

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org' do
   gem 'cocoapods', '1.0.0'
+  gem 'activesupport', '~> 4.0'
```
Comment (dfed) [inline Gemfile]: thanks @dnkoutso for providing this fix!

File: Sources/RemoteNotification.swift [hunk 2]
```diff
@@ -68,29 +66,39 @@ public struct RemoteNotification: CustomStringConvertible, Equatable {
...
+        /// The localization key for the title shown on the Apple Watch. Derived from userInfo["aps"]["alert"]["title-loc-key"]
         public let wearableTitleLocalizationKey: String?
-        /// The localization arguments for the title shown on the Apple Watch. Derived from userInfo["alert"]["title-loc-args"]
+        /// The localization arguments for the title shown on the Apple Watch. Derived from userInfo["aps"]["alert"]["title-loc-args"]
         public let wearableTitleLocalizationArguments: [String]?
         
         init?(remoteNotification: [String : AnyObject]) {
-            guard let alert = remoteNotification[alertKey] else {
-                return nil
+            guard let alert = remoteNotification[APSServiceKey]?[alertKey] as? [String : AnyObject] else {
```
Comment (pwesten) [inline Sources/RemoteNotification.swift]: Let's have a comment here explaining what's going on with this logic

**Comment by pwesten** [comment]
LGTM with the comment added in.

**Pull request merged**